packages/server/lib/modes/record.js

```diff
@@ -77,13 +77,14 @@ const throwIfIndeterminateCiBuildId = (ciBuildId, parallel, group) => {
   }
 }
 
-const throwIfRecordParamsWithoutRecording = (record, ciBuildId, parallel, group, tag) => {
-  if (!record && _.some([ciBuildId, parallel, group, tag])) {
+const throwIfRecordParamsWithoutRecording = (record, ciBuildId, parallel, group, tag, autoCancelAfterFailures) => {
+  if (!record && _.some([ciBuildId, parallel, group, tag, autoCancelAfterFailures || autoCancelAfterFailures === false])) {
```

Comment:
```suggestion
  if (!record && _.some([ciBuildId, parallel, group, tag, autoCancelAfterFailures !== undefined])) {
```

Reply:
If we want to throw always that the value exits, also when is not valid I think `_.some([ciBuildId, parallel, group, tag, autoCancelAfterFailures])`  may do the trick

Reply:
If we want to throw always that the value exits, also when is not valid I think `_.some([ciBuildId, parallel, group, tag, autoCancelAfterFailures])`  may do the trick